### DISCUSSION TITLE
Provide a way to set the "Save" and "Save As" location.

### DISCUSSION THREAD

**Discussion by robert-phillips-hpe** [brackets-cont/brackets#272] [Feature Request]
When saving a new file, selecting File - Save brings me to what I assume is the default "Getting Started" location.  I'd like a way to change the default behavior so I can set "Documents" or some other folder as my default saving directory.

**Comment by abose** [comment]
Thanks, @robert-phillips-hpe, for the feature request.
This will be fixed in the next phcode based release in June.
